### PR TITLE
Camera: Add autogenerated API client

### PR DESCRIPTION
**Pull request by kantoniak**

### CODE REVIEW

File: camera/rpi-files/setup.cfg [hunk 1]
```diff
@@ -3,9 +3,11 @@ multi_line_output=3
 include_trailing_comma=True
 force_grid_wrap=0
 use_parentheses=True
-line_length=88
+line_length=79
```
Comment (avan1235) [inline camera/rpi-files/setup.cfg]: what's this "magic number"

File: camera/rpi-files/setup.sh [hunk 2]
```diff
@@ -21,3 +21,7 @@ python3 -m venv env &&
 source env/bin/activate &&
 pip install -r requirements.txt &&
 deactivate
+
+# Download API generator
+wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/4.2.3/openapi-generator-cli-4.2.3.jar -O openapi-generator-cli.jar
```
Comment (avan1235) [inline camera/rpi-files/setup.sh]: maybe add string var whith configurable version number

File: camera/rpi-files/setup.cfg (hunk 1 above)
Comment (kantoniak) [reply camera/rpi-files/setup.cfg]: 79 is standard per PEP8 guidelines. I copied 88 from some article.

File: camera/rpi-files/setup.sh (hunk 2 above)
Comment (avan1235) [reply camera/rpi-files/setup.sh]: even specified one version for the whole projects as we are also going to use it in kotlin app

Comment (kantoniak) [reply camera/rpi-files/setup.sh]: I would keep separate copy for embedded devices, we don't know how to ship them yet.

Comment (avan1235) [reply camera/rpi-files/setup.sh]: But should it be like that every part of the system uses the same version of API generator java program to generate the needed classes etc.?

Comment (kantoniak) [reply camera/rpi-files/setup.sh]: Not necessarily, I think. API definition guarantees the contract, and clients just implement that.

Comment (kantoniak) [reply camera/rpi-files/setup.sh]: Not necessarily, I think. API definition guarantees the contract, and clients just implement that. E.g. camera doesn't care how Android implements the API and vice-versa.

Comment (kantoniak) [reply camera/rpi-files/setup.sh]: I moved generator version to a separate variable in this file, it's a good idea even if we don't share this value with other system parts.

**Comment by avan1235** [comment]
Approve but we have to think of using something to use it like env variable

**Pull request merged**